src/components/events/Events.tsx

```diff
@@ -75,13 +85,42 @@ const Events = () => {
         Error fetching events
       </p>
     );
-
   return (
     <div className="flex w-full flex-col font-anek-telegu text-3xl">
-      <div className="mt-8">
-        {events.map((event, index) => (
```

Comment:
Preferably we are still going to keep the mapping. 
Try to add the animation directly to the `Event` Component. Lmk if that doesn't work
Also, animate the calendar.